pkg/dnfjson/dnfjson.go

```diff
@@ -156,6 +156,13 @@ type Solver struct {
 	subscriptions *rhsm.Subscriptions
 }
 
+// DepsolveResult contains the results of a depsolve operation.
+type DepsolveResult struct {
+	Packages []rpmmd.PackageSpec
+	Repos    []rpmmd.RepoConfig
+	Sbom     *sbom.Document
```

Comment:
Thoughts on changing `Sbom` to `SBOM`?
I kinda expected it to be all caps since it's an initialism, like `RPM`, `URL`, `BIOS`...

Reply:
I have no strong preference. I can change it.

Reply:
renamed it to SBOM